bokehjs/src/coffee/renderer/tile/tile_renderer.coffee

```diff
@@ -24,6 +27,20 @@ class TileRendererView extends PlotWidget
     @y_range = @map_plot.get('y_range')
     @y_mapper = this.map_frame.get('y_mappers')['default']
     @extent = @get_extent()
+    @_add_attribution()
+
+  _add_attribution: () ->
+    if @mget('tile_source').get('attribution')?
+      div_check = Bokeh.$('.bk-canvas-overlays').children('#tile_attribution')
```

Comment:
Let me know when all functionality is on-board, then I will show you how to rewrite this, so that it'll be possible to merge this work.